src/components/select-custom/select-custom.component.html

```diff
@@ -0,0 +1,51 @@
+<button type="button"
+        class="select-button"
+        [uxMenuTriggerFor]="menu"
+        (click)="onToggle()">
+
+    <ng-container [ngTemplateOutlet]="buttonContentRef"></ng-container>
+
+    <div class="ux-select-icons">
+        <i class="ux-select-icon ux-icon ux-icon-close"
+            *ngIf="allowNull && selected"
+            (click)="resetValue($event)">
+        </i>
+
+        <i class="ux-select-icon ux-icon ux-icon-chevron-down ux-icon-down"></i>
```

Comment:
In the recent sprint we have went through our icon component to use instead of icon classes if they could be used here instead:

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_develop-1.8.2/#/components/icons#icon

Reply:
I fixed that. (To be honest, I had copied it from your select implementation.)